openapi-spec/swagger.yaml

```diff
@@ -87,7 +92,104 @@ tags:
     externalDocs:
       description: Find out more
       url: 'https://sodafoundation.io/'
+# schemes:
+# - http
 paths:
+  '/v1/{project_id}/aksks':
+    post:
+      tags:
+        - AKSK
+      summary: Create a new AKSK for a User.
+      operationId: addAKSK
+      parameters:
+        - name: project_id
+          in: path
+          required: true
+          description: AKSK to be generated for User of a Project
+          schema:
+            type: string
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AKSK'
+        '401':
+          description: Unauthorized. Token not found in header
+        '415':
+          description: Unsupported Media Type
+      requestBody:
+        $ref: '#/components/requestBodies/AKSK'
+  '/v1/{project_id}/aksks/{user_id}':
+    get:
```

Comment:
I am bit confused with the API,s . There no Create for user, but there is a GET for user.?

Reply:
Line no. 149 has delete API
Create API takes user_id in body 
{
    "credential": {
        "project_id": "94b280022d0c4401bcf3b0ea85870519",
        "user_id": "558057c4256545bd8a307c37464003c9"
    }
}
, GET takes user_id in the URL Param.  
Please let me know if any more confusion.